doc/api/vm.md

```diff
@@ -310,6 +310,27 @@ console.log(util.inspect(sandbox));
 // { globalVar: 1024 }
 ```
 
+## vm.moveMessagePortToContext(port, context)
```

Comment:
Maybe just `vm.transferMessagePort()`? The "to context" part is already conveyed by the fact that this method is in `vm` module, and "transfer" is a more Proper(R) name IMO than "move".

Reply:
What I like about `move` rather than `transfer` is that it conveys better that the original message port object is now rendered unusable … I don’t feel strongly about it, though